sa/sa.go

```diff
@@ -1357,6 +1359,10 @@ func (ssa *SQLStorageAuthority) PauseIdentifiers(ctx context.Context, req *sapb.
 				// Identifier is already paused.
 				continue
 
+			case entry.UnpausedAt != nil && entry.UnpausedAt.After(ssa.clk.Now().Add(-14*24*time.Hour)):
```

Comment:
Because we've already passed `case entry.UnpausedAt == nil`, we don't need to check it again here.
```suggestion
			case entry.UnpausedAt.After(ssa.clk.Now().Add(-14*24*time.Hour)):
```